src/esw/brushed_motors/Controller.cpp

```diff
@@ -40,6 +39,25 @@ float Controller::getCurrentAngle() const {
     return currentAngle;
 }
 
+// REQUIRES: newAngleRad to be in radians
+// MODIFIES: currentAngle
+// EFFECTS: I2C bus, forces the angle of the controller to be a certain value
+void Controller::overrideCurrentAngle(float newAngleRad) {
+    auto ticks = (int32_t) (((newAngleRad) / (2 * M_PI)) * quadCPR); // convert to quad units
```

Comment:
Is there a reason for this to be auto and not just int32_t?

Reply:
Type is already declared in a cast, no need for redundancy. Clang also wants it to be like that.